superset-frontend/packages/superset-ui-chart-controls/src/operators/flattenOperator.ts

```diff
@@ -19,9 +19,7 @@
 import { PostProcessingFlatten } from '@superset-ui/core';
 import { PostProcessingFactory } from './types';
 
-export const flattenOperator: PostProcessingFactory<PostProcessingFlatten> = (
-  formData,
-  queryObject,
-) => ({
-  operation: 'flatten',
-});
```

Comment:
The operators in AA are intentionally designed to be a unified interface. These operators may use `formData` or `queryObject` at any time.